composer.json

```diff
@@ -5,20 +5,12 @@
     "license": "Apache-2.0",
     "authors": [
       {
-        "name": "Nils Destoop",
-        "email": "nils.destoop@wunderkraut.com"
-      },
-      {
-        "name": "Thomas Jacobs",
-        "email": "thomas.jacobs@wunderkraut.com"
-      },
-      {
-        "name": "Stijn Swaanen",
-        "email": "stijn.swaanen@publiq.be"
+        "name": "Publiq vzw",
+        "email": "info@publiq.be"
       }
     ],
     "require": {
-      "php": "^7.0",
+      "php": "^7.1",
```

Comment:
Let's hope no one from other companies is using this on their project running 7.0 🤞